### DISCUSSION TITLE
Remix IDE deploying to zkSync L2

### DISCUSSION THREAD

**Discussion by vpanevv** [Cyfrin/foundry-full-course-cu#3077] [Q&A] [answered]
Hi all, I have trouble deploying my first contract on L2 zkSync from Remix IDE. I do everything from the course but still get the same error: 

<img width="1340" alt="Screenshot 2024-12-19 at 22 56 03" src="https://github.com/user-attachments/assets/79185eb4-07b0-4d42-9886-8d931bd071d9" />

Will be happy if someone can help me with this.

**Comment by EngrPips** [comment]
Hello @vladimirpanev95, Please uncheck the `verify contract` box and retry the deployment and let's see if it will work.

**Reply by vpanevv** [reply]
Hi @EngrPips, thanks but still the same error appear.

**Reply by EngrPips** [reply]
Tough. Now clear your browser cache and retry, and let's see if that helps. It also appears to me that your Metmask is on `sepolia` chain and not `sepolia-zksync` chain, Please change that before you retry.

**Reply by Fallin-1-dev** [reply]
Hello. Have you been able to resolve it?
I am also having same problem right now.

**Reply by EngrPips** [reply]
Hey @Fallin-1-dev, are you sure your Metamask is switched `zksync-sepolia` test net and not `sepolia` test net?

**Reply by carlosduardo** [reply, marked as answer]
Create a directory(box) and put the file.sol inside. Chexk the current Solidity's version in the file 0.8.28

**Reply by vpanevv** [reply]
@carlosduardo  Yes, that fixed the problem. I moved all the contracts in one folder. Thanks!

**Reply by EngrPips** [reply]
Awesome.
